### PR TITLE
Fixed issue in download schema at store

### PR DESCRIPTION
**Pull request by HiranyaKavishani**

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx [hunk 1]
```diff
@@ -542,6 +543,24 @@ class InfoBar extends React.Component {
...
+        const { api } = this.context;
+        const promisedGraphQL = newAPI.getGraphQLSchemaByAPIId(apiId);
+        promisedGraphQL.then((response) => {
+            const windowUrl = window.URL || window.webkitURL;
+            const binary = new Blob([response.data]);
+            const url = windowUrl.createObjectURL(binary);
+            const anchor = document.createElement('a');
+            anchor.href = url;
+            anchor.download = api.provider + '-' + api.name + '-' + api.version
+            + '.graphql';
```
Comment (TanyaM) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx]: can go in one line

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx [hunk 2]
```diff
@@ -670,12 +682,13 @@ class InfoBar extends React.Component {
                                                             </div>
                                                         </TableCell>
                                                         <TableCell>
-                                                            <a className={classes.downloadLink} href={downloadLink} download={fileName}>
+                                                            <Button onClick={this.getSchema} size='small' fontSize='small' variant='outlined'>
```
Comment (TanyaM) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx]: looks like exceeding 120 line break. Please check.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx (hunk 1 above)
Comment (HiranyaKavishani) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx]: Fixed with ead77f9051a2e0c270afb84bd6c21f3263995bbe

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx (hunk 2 above)
Comment (HiranyaKavishani) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx]: Fixed with ead77f9051a2e0c270afb84bd6c21f3263995bbe

**Pull request merged**